python/lsst/analysis/tools/atools/photometry.py

```diff
@@ -86,9 +86,9 @@ def setDefaults(self):
         self.produce.metric.units = {"median": "mmag", "sigmaMad": "mmag", "mean": "mmag"}
 
         self.produce.metric.newNames = {
-            "median": "{band}_ap12-psf_median",
-            "mean": "{band}_ap12-psf_mean",
-            "sigmaMad": "{band}_ap12-psf_sigmaMad",
+            "median": "{band}_ap12_diff_psf_median",
+            "mean": "{band}_ap12_diff_psf_mean",
+            "sigmaMad": "{band}_ap12_diff_psf_sigmaMad",
```

Comment:
If we're keeping this in-line with the prior renaming exercise on DM-42062, these should be:
```
            "median": "{band}_ap12_psf_diff_median",
            "mean": "{band}_ap12_psf_diff_mean",
            "sigmaMad": "{band}_ap12_psf_diff_sigmaMad",
```
I.e., the Ap12 and PSF difference.